### PR TITLE
Bot config

### PR DESCRIPTION
**Pull request by Anita-ihuman**
This PR fixes issue #277 
### Description
Created configuration file for the Behaviour bot

### CODE REVIEW

File: content/en/docs/Releases/_index.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title:  "Release"
+weight: 55
+description:  Kyverno Version Release Notes.
+---
+
+## Kyverno v1.4.2
```
Comment (chipzoller) [inline content/en/docs/Releases/_index.md:7]: As mentioned, there are really quite a lot of other bullet points to mention for each release. Not sure if you intended to cover those in a separate PR at a later time though.

Comment (Anita-ihuman) [reply content/en/docs/Releases/_index.md:7]: This PR was intended to get more feedback regards what the release note should look like.

Comment (Anita-ihuman) [reply content/en/docs/Releases/_index.md:7]: I am currently using the sample you provided to update this Document.

**Comment by Anita-ihuman** [comment]
@chipzoller I am still updating the release notes, but then I observed that every PR i create now, tags along with the Release PR. Even when i create a seperate branch for it.

**Pull request closed without merging**